app/models/miq_alert.rb

```diff
@@ -193,16 +193,22 @@ def evaluate(target, inputs = {})
     # If we are alerting, invoke the alert actions, then add a status so we can limit how often to alert
     # Otherwise, destroy this alert's statuses for our target
     invoke_actions(target, inputs) if result
-    add_status_post_evaluate(target, result, inputs[:description])
-
+    add_status_post_evaluate(target, result, inputs[:ems_event])
```

Comment:
Can you extract this change into a different commit so info is retained on the change?